### PR TITLE
Fix test discrepancy

### PR DESCRIPTION
**Pull request by JosiahRegencia**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -14,8 +14,9 @@ services:
       timeout: 30s
       retries: 3
     # UN-COMMENT for local testing:
-    #ports:
-    #  - "5432:5432"
+    env_file: .env
```
Comment (greenmoss) [inline docker-compose.yml]: We have to leave this commented

File: .travis.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ sudo: true
 before_script:
   - docker-compose up -d
   - ./db/ready.sh
-script: docker exec mono_savant_1 python tests/test_basic.py
+script: docker exec mono_savant_1 sh tests/db.sh
```
Comment (greenmoss) [inline .travis.yml]: should this stay as the original?

File: savant/app/get_scanned.py [hunk 3]
```diff
@@ -21,19 +23,22 @@ def parse(scanner_directory):
...
 
-        elif "shadow" in str(filename.filename):
+        elif "shadow" in str(filename):
             get_shadow(filename)
 
-        elif "users" in str(filename.filename):
+        elif "users" in str(filename):
             get_users(filename)
 
+    logging.debug('\nBeen to get_items')
```
Comment (greenmoss) [inline savant/app/get_scanned.py]: better to remove `\n` from logging statements

File: savant/app/get_scanned.py [hunk 4]
```diff
@@ -42,12 +47,15 @@ def replace_blank(parts):
     return parts
 
 
-
 def get_debs(filename):
+    logging.debug('\n\File: {}\n\n'.format(filename))
```
Comment (greenmoss) [inline savant/app/get_scanned.py]: for log statements, use commas for arguments

`debug('File: %s', filename)`

File: savant/app/get_scanned.py [hunk 5]
```diff
@@ -58,12 +66,17 @@ def get_debs(filename):
         parts[0:4] = replace_blank(parts[0:4])
         debs.append(parts[0:4])
 
+    ### Shows the results of the scanned inputs ###
+    logging.debug('\nDebs: {}'.format(debs))
```
Comment (greenmoss) [inline savant/app/get_scanned.py]: same thing for commas here

File: savant/app/get_scanned.py [hunk 6]
```diff
@@ -74,12 +87,17 @@ def get_groups(filename):
         parts[0:4] = replace_blank(parts[0:4])
         groups.append(parts[0:4])
 
+    ### Shows the results of the scanned inputs ###
+    logging.debug('\nGroups: {}'.format(groups))
```
Comment (greenmoss) [inline savant/app/get_scanned.py]: commas again

File: savant/app/get_scanned.py [hunk 7]
```diff
@@ -4,6 +4,8 @@
 import logging.config
 # from app import record
 
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s : %(levelname)s : %(message)s')
```
Comment (greenmoss) [inline savant/app/get_scanned.py]: default log level should be WARNING

File: savant/app/query.py [hunk 8]
```diff
@@ -9,6 +9,8 @@
 class Query:
     def __init__(self):
 
+        logging.basicConfig(level=logging.DEBUG, format='%(asctime)s : %(levelname)s : %(message)s')
```
Comment (greenmoss) [inline savant/app/query.py]: `WARNING` ;)

**Comment by JosiahRegencia** [comment]
I'm getting this from travis:
Error response from daemon: Container 9971df3bed5bb2bb7870d98ac81c2a6293928c94b40695880a643d81e3501844 is not running

screenshot from 2018-04-05 22-22-38

**Comment by JosiahRegencia** [comment]
https://travis-ci.org/Doveps/mono/builds/362662427

File: savant/app/query.py [hunk 9]
```diff
@@ -153,3 +157,39 @@ def new_users(self):
         self.conn.commit()
         
         return self.new_users
+
```
Comment (greenmoss) [inline savant/app/query.py]: Following up with the meeting last Wednesday:

Please transition testing to use only `GET` and `CREATE` API endpoints for the testing. Going that route, I am guessing we don't need `null_cases` or `count_rows`. The test instead does a `CREATE` that would have triggered the null condition, then does a `GET` to verify the count of records.

**Review by greenmoss — CHANGES_REQUESTED** [review]

**Comment by JosiahRegencia** [comment]
screenshot from 2018-04-12 18-45-49
 PASSED!
@greenmoss

**Comment by greenmoss** [comment]
Excellent!

To merge this, need:
* fix conflicts
* update to only `GET`/`CREATE`, as per review

**Comment by JosiahRegencia** [comment]
Fixed conflicts @greenmoss

**Pull request closed without merging**